Test/Unit/Model/RunnerTest.php

```diff
@@ -48,15 +49,24 @@ protected function setUp(): void
             ]
         );
 
+        $this->emailTemplateChecker = $this->createMock(EmailTemplateChecker::class);
+
+        $mockOutput = [
```

Comment:
I wanted to mock the `mapEmailTemplates` in the test, but that did not work for me

So, i had to mock the actual `check` method